module/input/SensorFilter/README.md

```diff
@@ -16,8 +16,10 @@ The resulting filtered data will then be outputted as `message::input::Sensors`
 ## Emits
 
 - `message::input::Sensors` with filtered data from the input.
--
+- `message::platform::ButtonLeftDown` when button pressed.
+- `message::platform::ButtonLeftUp` when button released.
+- `message::platform::ButtonMiddleDown` when button pressed.
+- `message::platform::ButtonMiddleUp` when button released.
+- `message::support::nusight::DataPoint` if log_level >= DEBUG, indicating if feet are down.
```

Comment:
I think this is a `utility::nusight::graph`? not data point

Reply:
`graph` is a [helper function that creates and emits DataPoint](https://github.com/NUbots/NUbots/blob/c550271424d0c7742069e87dac371a49c1b0c5b6/shared/utility/nusight/NUhelpers.hpp#L60).